src/libraries/System.Reflection.Metadata/src/System/Reflection/Internal/MemoryBlocks/MemoryMappedFileBlock.cs

```diff
@@ -18,10 +17,6 @@ private sealed class DisposableData : CriticalDisposableObject
 
             public DisposableData(IDisposable accessor, SafeBuffer safeBuffer, long offset)
             {
-                // Make sure the current thread isn't aborted in between acquiring the pointer and assigning the fields.
-#if !NETSTANDARD1_1
-                RuntimeHelpers.PrepareConstrainedRegions();
-#endif
```

Comment:
@tmat, I assume the netstandard2.0 build would still want these for .NET Framework? If yes, all of the changes under System.Reflection.Metadata should be reverted.

Reply:
Yes, I would think so.